### PR TITLE
docs: Update install guide for fedora

### LINKED ISSUE #218: Update install guide for Fedora
Update install documentation guide for fedora to include the support of fedora 28.

### PR DESCRIPTION
**Pull request by GabyCT**
Update install documentation guide for fedora to include the
support for fedora 28.

Fixes #218

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by ydjainopensource** [comment]
lgtm

File: install/fedora-installation-guide.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 > **Notes:**
 >
 > - Kata Containers packages are available for [Fedora\*](https://fedoraproject.org)
->   versions **26** and **27** (currently `x86_64` only).
+>   versions **26**, **27** and **28** (currently `x86_64` only).
```
Comment (jodh-intel) [inline install/fedora-installation-guide.md]: Shouldn't we remove mention of F26 as it's now EOL?

- https://en.wikipedia.org/wiki/Fedora_version_history
- https://fedoraproject.org/wiki/End_of_life

If so, we'll also need to remove the F26 OBS packages and config too..

**Comment by GabyCT** [comment]
@jodh-intel changes applied

**Comment by jodh-intel** [comment]
Thanks @GabyCT.

lgtm

**Comment by caoruidong** [comment]
lgtm

**Pull request merged**